src/main/java/bsu/fpmi/glitter/servlets/tweets/TweetServlet.java

```diff
@@ -0,0 +1,185 @@
+package bsu.fpmi.glitter.servlets.tweets;
+
+import bsu.fpmi.glitter.Tweet;
+import bsu.fpmi.glitter.TweetContainer;
+import bsu.fpmi.glitter.TweetFilterConfig;
+
+import javax.servlet.ServletException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class TweetServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        resp.setContentType("application/json");
+        resp.setCharacterEncoding("UTF-8");
+
+        String id = req.getParameter("id");
+
+        Tweet tweet = TweetContainer.localDatabase.get(id);
+        String str;
+        if(tweet == null)
+            str = "{\n}\n";
```

Comment:
Adding `\n` here is senseless. Browser will not be much more impressed than if you sent `{}`)